.cbmc-batch/jobs/aws_cryptosdk_keyring_on_encrypt/aws_cryptosdk_keyring_on_encrypt_harness.c

```diff
@@ -89,6 +89,7 @@ void aws_cryptosdk_keyring_on_encrypt_harness() {
                                                      .on_decrypt = nondet_voidp() };
     struct aws_cryptosdk_keyring keyring;
     ensure_cryptosdk_keyring_has_allocated_members(&keyring, &vtable);
+    __CPROVER_assume(keyring.refcount.value != NULL);
```

Comment:
Why not assume the invariant here? What is the equivalence precondition in the harness?

Reply:
Same as above!